src/vs/editor/browser/services/openerService.ts

```diff
@@ -116,7 +116,9 @@ export class OpenerService implements IOpenerService {
 		// Default opener: maito, http(s), command, and catch-all-editors
 		this._openers.push({
 			open: async (target: URI | string, options?: OpenOptions) => {
-				if (options?.openExternal || matchesScheme(target, Schemas.mailto) || matchesScheme(target, Schemas.http) || matchesScheme(target, Schemas.https)) {
+				// {{SQL CARBON EDIT}} - Begin
```

Comment:
can we just pass in the openExternal option from the caller side where we need this? Would there be anywhere else besides notebooks where this is applicable?

Reply:
You shouldn't need these edits, it checks for the openExternal field first. just call the function with openExternal options as true.

Reply:
Thanks for the tip. Sent an update which limits the change to the linkHandler directive, and affects the Onenote links only.